modules/views/civicrm.views.inc

```diff
@@ -304,6 +304,9 @@ function civicrm_views_add_date_arguments(&$data, $value) {
  *   Array with the new custom field appended
  */
 function civicrm_views_custom_data_cache(&$data, $entity_type, $group_id, $sub_type) {
+  if (!CRM_Core_Config::isUpgradeMode()) {
```

Comment:
@KarinG @eileenmcnaughton i have just changed it to remove the ! as i tested it and didn't seem to make a difference and i think is the actual correct thing we should be testing